Frontend/Pages/TaskList/HtbDocument/Preview.cshtml.cs

```diff
@@ -125,7 +126,25 @@ public async Task<IActionResult> OnGet(string id)
                 ReturnToPreview = true
             };
 
-            PupilNumbersViewModel = PupilNumbersController.BuildViewModel(response,true, true);
+            PupilNumbersViewModel = new PupilNumbers(_getInformationForProject, _projects)
```

Comment:
The additional information and change link doesn't appear to show on the preview page. May need to set that IsPreview to true possibly